pages/common/calendar.md

```diff
@@ -0,0 +1,32 @@
+# calendar
+
+> Display upcoming events from a calendar file.
+> Basic event format: "(month)/(day)    (Description)" (tab-separated).
+
+- Show events for today and tomorrow (or the weekend on Friday) from the default calendar:
+
+`calendar`
+
+- Look [A]head, showing events for the next 30 days:
+
+`calendar -A {{30}}`
+
+- Look [B]ack, showing events for the previous 7 days:
+
+`calendar -B {{7}}`
+
+- Show events from a custom calendar [f]ile:
+
+`calendar -f {{path/to/file}}`
+
+- Example entry for Christmas:
```

Comment:
We don't include output examples in the tldr pages. Only examples for the command usage.

Reply:
Those are actually examples of how to add custom events if that makes a difference. I can take them out (or clarify) if you want it to be just command line arguments.

Reply:
So, should we add events with this format to get processed by the app? :thinking: Maybe a `tee` command appending events to the calendar file?

Reply:
No, as far as I know you have to edit the file manually. The calendar app just reads it. This discussion is actually starting to convince me that it doesn't make sense to have the file format info in tldr, since it's hard to describe how it works concisely. The man page has examples if people need them.

Reply:
@silentmatt, please apply the changes so I can re-check.